docs/devops/dns.md

```diff
@@ -1,5 +1,25 @@
-Our DNS records are handled by CloudFlare.
+Our DNS records are handled by Cloudflare.
 
 ## Adding a new record
 
-todo
\ No newline at end of file
+To add a new record, go to Cloudflare and add a record.
+
+![cloudflare](img/cloudflare.png)
+
+## Page Rules
+
+We have some custom page rules set up on Cloudflare as well, which allow us to cache specific endpoints or create custom redirects.
+
+For example, we have a custom page rule used to cache the API endpoint to get the list of all hacks from Gavel, for performance reasons.
+
+## Warning about root CNAME records
```

Comment:
@epicfaace seeing this written up is so funny and yet so sad 😢 
postmortems got me like...